### DISCUSSION TITLE
Do KNI content pipeline and mgcb editor run on arm-based MacOS?

### DISCUSSION THREAD

**Discussion by kurtcz** [kniEngine/kni#2012] [Content Pipeline]
I develop an Android/iOS game running on MonoGame and have read that sadly MG does not have a good support for Apple Sillicon based macs - see [here](https://community.monogame.net/t/tutorial-for-setting-up-monogame-on-m1-m2-apple-silicon/19669). I plan to buy a new mac soon (my current machine is already very old) and would like to know: 
- if the KNI suffers from the same issues or not. 
- Does the content pipeline incl. editor work on an arm-based mac without any workarounds? 
- Can it run without an x64 .NET SDK installed?
 
Many thanks for the answer.

**Comment by nkast** [comment]
Hi!
The content builder and the editor doesn't support mac nor linux.
The focus is on windows as the dev enviroment.
What is missing is the native libraries that are used my the stock importer/processors. If anyone want to put some effort into building those libraries for mac-arm and include them in the contentBuilder I wouldn't be against merging the changes.

**Reply by bosoni** [reply]
>  I would be against merging the changes.

I think you meant you would not be against merging the changes

**Reply by nkast** [reply]
Yeah!
*Edited*
